wagtail/admin/views/generic/history.py

```diff
@@ -130,11 +131,7 @@ def get_index_results_url(self):
     def get_context_data(self, *args, object_list=None, **kwargs):
         context = super().get_context_data(*args, object_list=object_list, **kwargs)
         context["object"] = self.object
-        context["header_action_url"] = self.get_edit_url(self.object)
-        context["header_action_label"] = gettext("Edit this %(model_name)s") % {
-            "model_name": self.model._meta.verbose_name
-        }
-        context["header_action_icon"] = "edit"
+        context["model_opts"] = BaseLogEntry._meta
```

Comment:
The results template still uses `index_results.html` but now without the `no_results_message` block override, so we need this in order to show `"There are no {{ model_name }} to display."`

We can also use the queryset's `.model._meta`, but that means it'll read as `"There are no model log entries to display"` which seems too specific for end-users, especially for models with custom log entry models. With the `BaseLogEntry`, it just shows as `"There are no log entries to display."`, which is quite similar to the previous `no_results_message` block override.